bin/interop-decode.c

```diff
@@ -47,6 +47,12 @@ static int s_verbose;
 
 static FILE *s_out;
 
+static unsigned
+min(unsigned a, unsigned b)
```

Comment:
I'd make it a macro -- just copy one of the several `MIN` definitions in the project.

Reply:
...and the Windows build apparently fails here